test.js

```diff
@@ -210,6 +211,24 @@ test('sync (cousin file, custom cwd)', t => {
 	t.is(foundPath, absolute.baz);
 });
 
+test('async (cousin file, custom cwd with stopAt)', async t => {
+	const foundPath = await findUp(name.baz, {
+		cwd: relative.barDir,
+		stopAt: absolute.fooDir,
+	});
+
+	t.is(foundPath, undefined);
+});
```

Comment:
Can you also do a test where the foundPath is the `stopAt` path?

Reply:
Yep, added:
https://github.com/sindresorhus/find-up/pull/54/commits/d2a1b0d6f7fbc6371314b096be4fdf82ddff0956